### ISSUE TITLE
Failed to `yarn build:analysis` at `./packages/sandbox/`

### ISSUE THREAD

**Issue by xiongmao86** [epicmaxco/vuestic-ui#3296]
**Vuestic-ui version:** 1.6.3

**Steps to reproduce** ~~`yarn build:docs`~~ `yarn build:analysis` at ~~`./packages/docs`~~ `packages/sandbox`

**What is the expected behavior?** build should success.

**What is the current behavior?** build failed with `Error: File './src/.nuxt/tsconfig.json' not found.`

**Other information**
I waited for a couple days, no body seems to have the same issue with me. Please ping me if anything is needed. Thanks.

**Comment by rustem-nasyrov** [comment]
Hello, @xiongmao86! I checked your repository fork and it seems to be outdated. 
1. Open your vuestic-ui' repository fork.
2. Click on "Sync fork" dropdown.
3. Click on "Update branch".

<img width="908" alt="image" src="https://user-images.githubusercontent.com/29167241/229541944-40664549-ca7b-460c-ae33-5b3d1cbd3108.png">

**Comment by rustem-nasyrov** [comment]
Also, try to install all dependencies with `yarn --force`. After that, regenerate your `.nuxt` folder with following instructions:
1. If you at root directory of the project, checkout to the `packages/docs` directory.
2. Run `npx nuxi prepare` command.

**Comment by xiongmao86** [comment]
Thanks very much for your instruction, @rustem-nasyrov, I have update to the most recent commit and have tried to install all dependencies with `yarn --force`, and also ran `npx nuxi prepare` in the `packages/docs` and still have the same output, `./src/.nuxt/tsconfig.json` is still missing.

I have try `yarn build:analysis` at `./packages/docs`, that seems to be the exact location of the error.
<img width="650" alt="build analysispng" src="https://user-images.githubusercontent.com/1770218/229692655-293a55ce-f869-4f62-920a-736605da4c55.png">

**Comment by xiongmao86** [comment]
I have taken a closer look today. Never expect to learn, just to be more familiar. It turns out there are something to be learned. 

The file `./src/.nuxt/tsconfig.json` is make by `packages/sandbox/script/stud-nuxt.mjs`. 

I am not sure changing it is the proper way to do. So I just take a shoot and see what would happen. There is another error in `vite build --config ./configs/vite/vite.button.ts`. 

That could probably because of an import appeared inside an intermediate file `VaModal-96ddc65e.mjs` of component `va-modal`. This is that import statement: `import '..\..\../VaModal.css'`. I have checked that escape character works in single quoted string literal. So I guess it's probably result in `import '....../VaModal.css'`, which I think would error out. Is that the exact problem? I am not very confident at this theory.

**Comment by m0ksem** [comment]
Hey, @xiongmao86. I assume you're on a Windows machine. I'll try to reproduce it and see what we can do here. 
Until then, you can try locally change line 37 in `packages/ui/build/plugins/append-component-css.ts` 
from 
```ts
const relativeFilePath = relativeDistPath + '/' + name.replace(/-.*$/, '') + '.css'
```
to 
```ts
const relativeFilePath = resolve(relativeDistPath, name.replace(/-.*$/, '') + '.css')
```
and see if it resolves issue with importing VaModal.css.

**Comment by xiongmao86** [comment]
Thank you very much, @m0ksem, I am actually on a Windows machine. I have check it out today, doesn't seem to be working. I also notice I have a global link to an older version of `vite` in global. Should I first work out how to remove the link, or should I first try to look into the error message with the older version of `vite`?

**Comment by m0ksem** [comment]
@xiongmao86, please, try https://github.com/epicmaxco/vuestic-ui/pull/3311 and see if it fixes this issue. Likely, you will need a fresh setup.

**Comment by xiongmao86** [comment]
Sorry, @m0ksem, I'll follow your guide. I have `git pull` your `fix/windows-docs-serve` branch and `yarn` and testing with `yarn build:analysis`, the terminal console bailed an error:

> Error  vite build --config ./configs/vite/vite.button-select.ts exited with code 1
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "vite v4.2.1 building for production...
transforming...
✓ 1 modules transformed.
✓ built in 45ms
".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

After that I have tried `npx vite build --config ./config/vite/vite.button-select.ts`, the console bailed out this message:
> vite v4.2.1 building for production...
✓ 1 modules transformed.
✓ built in 45ms
[commonjs--resolver] Failed to resolve entry for package "vuestic-ui". The package may have incorrect main/module/exports specified in its package.json.
error during build:
Error: Failed to resolve entry for package "vuestic-ui". The package may have incorrect main/module/exports specified in its package.json.
    at packageEntryFailure (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/vite/dist/node/chunks/dep-79892de8.js:23266:11)
    at resolvePackageEntry (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/vite/dist/node/chunks/dep-79892de8.js:23263:5)
    at tryNodeResolve (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/vite/dist/node/chunks/dep-79892de8.js:23000:20)
    at Object.resolveId (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/vite/dist/node/chunks/dep-79892de8.js:22762:28)
    at file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/rollup/dist/es/shared/node-entry.js:24343:40
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async PluginDriver.hookFirstAndGetPlugin (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/rollup/dist/es/shared/node-entry.js:24243:28)
    at async resolveId (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/rollup/dist/es/shared/node-entry.js:23187:26)
    at async ModuleLoader.resolveId (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/rollup/dist/es/shared/node-entry.js:23451:15)
    at async Object.resolveId (file:///C:/Users/xiongmao86/projects/vuestic-ui/packages/tailwind/node_modules/vite/dist/node/chunks/dep-79892de8.js:8006:10)

**Comment by xiongmao86** [comment]
I am sorry @m0ksem, does no repro means you can't repro in windows, or in other system you guys use to develop Vuestic-ui?

**Comment by Sadnessa** [comment]
Well, I have the same problem. But if you use  `yarn serve:docs`, it works just fine. x)

**Comment by xiongmao86** [comment]
@Sadnessa, thank you, I'll try that out.

**Comment by m0ksem** [comment]
I think this issue can be closed after #3296 and #3311 because both build and serve work on windows.

**Comment by xiongmao86** [comment]
Thank you very much, @m0ksem.
